projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab-step/lab-step.component.scss

```diff
@@ -9,6 +9,20 @@
       code {
         white-space: pre-wrap;
       }
-    }
   }
 }
+}
+
+img{
```

Comment:
I think we should add a class to the img component in the dialog and then use it here, will increase the specificity